combat-trainer.lic

```diff
@@ -1853,12 +1853,14 @@ class ManipulateProcess
     @manip_to_train = settings.manipulate_only_to_train
     echo("  @manip_to_train: #{@manip_to_train}") if $debug_mode_ct
 
+    Flags.add('ct-manip-broke', 'You feel your link with .* suddenly break', 'You feel your link suddenly break', 'You feel your link with .* weakening with distance.')
+
     @last_manip = Time.now - 200
   end
 
   def execute(game_state)
     return if game_state.danger || @threshold.nil? || game_state.construct_mode?
-    @filtered_npcs = game_state.npcs
+    @filtered_npcs = DRRoom.npcs#game_state.npcs
```

Comment:
❓ This doesn't look right to me. Why was this changed?

Reply:
if your hunting constructs or undead in a room that has live npcs, you need to put the live npcs in the ignore npc list. Without this change you won't manipulate the live npcs.